subprojects/sno-snapshot/src/snapshot.py

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env python
+
+import sys
+
+import common
+
+client_ec2 = common.client_ec2
+resource_ec2 = common.resource_ec2
+client_k8s = common.client_k8s
+
+def create_snapshot(cluster_name, instance, zone):
+    volume = common.get_instance_root_volume(instance)
+
+    print(f"Volume Id: {volume.id}")
+
+    ts = common.human_ts()
+
+    snapshot = volume.create_snapshot(
+        Description=f'Snapshot of cluster {cluster_name}.',
+        TagSpecifications=[{
+            'ResourceType': 'snapshot',
+            'Tags': [
+                {'Key': 'Name', 'Value': f'{cluster_name}-{ts}-snapshot'},
```

Comment:
I admit I'm heavily influenced by (the beauty of) ZFS, so I'd like to see something like `@{ts}` instead of this for a snapshot, but your preference may vary.

Reply:
hum, `{cluster_name}` must remain, because the names are for the whole AWS account, so better have them fully qualified
the `-snapshot` suffix can go away, as it's already the resource type

Reply:
I didn't want to get rid of the `{cluster_name}`, that's fine.  Just the `-snapshot` seemed a bit redundant.